site/content/contribute/more-info/mobile/developer-setup/_index.md

```diff
@@ -8,207 +8,250 @@ aliases:
   - /contribute/mobile/developer-setup
 ---
 
-The following instructions apply to the mobile apps for iOS and Android built in React Native. Download the iOS version {{< newtabref href="https://apps.apple.com/us/app/mattermost/id1257222717" title="here" >}} and the Android version {{< newtabref href="https://play.google.com/store/apps/details?id=com.mattermost.rn" title="here" >}}. Source code can be found at the {{< newtabref href="https://github.com/mattermost/mattermost-mobile" title="GitHub Mattermost Mobile app repository" >}}.
+The following instructions apply to the mobile apps for iOS and Android built in React Native.
+Download the iOS version {{< newtabref href="https://apps.apple.com/us/app/mattermost/id1257222717" title="here" >}} and the Android version {{< newtabref href="https://play.google.com/store/apps/details?id=com.mattermost.rn" title="here" >}}.
+Source code can be found at the {{< newtabref href="https://github.com/mattermost/mattermost-mobile" title="GitHub Mattermost Mobile app repository" >}}.
 
-If you run into any issues getting your environment set up, check the {{< newtabref href="https://docs.mattermost.com/deploy/mobile-troubleshoot.html" title="Troubleshooting" >}} section in the left sidebar for common solutions.
-
-{{<note "Note:">}}
-**This guide describes how to set up the development environment on macOS or Linux.**
-{{</note>}}
+If you run into any issues getting your environment set up, check the {{< newtabref href="https://docs.mattermost.com/deploy/mobile-troubleshoot.html" title="Troubleshooting" >}} section of the product docs for common solutions.
```

Comment:
in this link I see
> First, confirm that your server URL has no typos and that it includes http:// or https:// according to the server deployment configuration.

There is no need to include the protocol.

> [I see a “Connecting…” bar that does not go away](https://docs.mattermost.com/deploy/mobile-troubleshoot.html#contents)

The bar no longer says connecting... @larkox can you provide the correct text being displayed?

> [None of these solve my problem!](https://docs.mattermost.com/deploy/mobile-troubleshoot.html#contents)

in this section there is a step to use `demo.mattermost.com` that server does not longer exists.

Reply:
These comments apply to a document outside of developers.mattermost.com.
@cwarnermm I think there needs to be a ticket posted to fix these up in the product docs.

Reply:
Thanks, @neflyte. I created [a card](https://community.mattermost.com/boards/team/rcgiyftm7jyrxnma1osd8zswby/bbusacfknofye5jgtmrufo75mnc/v5hakifb3i7nw5emhjtbrdua7uy/cuwo9wu66j3fybgtm1izuqr9zno) to investigate further and update the docs where needed.